### ISSUE TITLE
address handleOfferedHashesMsg timeout errors

### ISSUE THREAD

**Issue by nonsense** [ethersphere/swarm#862]
We are getting a lot of timeout errors on the `open` cluster related to `handleOfferedHashesMsg`. We should review them and either fix them, or downgrade them from ERROR if these are expected scenarios.

```
swarm_30400 ERROR[08-08|10:08:26.901|swarm/network/stream/messages.go:240] client.handleOfferedHashesMsg() timeout waiting for batchDone, dropping peer ctx.Err()="context deadline exceeded"
swarm_30401 ERROR[08-08|10:08:26.952|swarm/network/stream/messages.go:228] client.handleOfferedHashesMsg() error waiting for chunk err="context deadline exceeded"
swarm_30401 ERROR[08-08|10:08:26.952|swarm/network/stream/messages.go:231] client.handleOfferedHashesMsg() timeout waiting for chunk, dropping peer ctx.Err()="context deadline exceeded"
```

cc @zelig @gbalint @janos

**Comment by nonsense** [comment]
This is from the `chunk-refactor-rebased` branch though, not `master`, so I am not sure this should be addressed in the 0.3.2 milestone.

**Comment by nonsense** [comment]
Looks like this has been resolved in the `chunk-refactor-rebased` branch.
